hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestStreamingSource.scala

```diff
@@ -205,13 +205,10 @@ class TestStreamingSource extends StreamTest {
       addData(tablePath, Seq(("2", "a1", "11", "001")))
       addData(tablePath, Seq(("3", "a1", "12", "002")))
 
-      val timestamp = if (handlingMode == USE_TRANSITION_TIME) {
+      val timestamp =
```

Comment:
Similar here.  Should we add a test case where the smallest completion time in the active timeline does not have the smallest instant time to validate the correctness of the incremental query logic?

Reply:
Filed JIRA: https://issues.apache.org/jira/browse/HUDI-8358